api/internal/core/store/selector_test.go

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
??????

Reply:
Is this test case file copy from other repo?

Reply:
some code from:
https://github.com/kubernetes/dashboard/tree/master/src/app/backend/resource/dataselect
not whole file.